cdap-watchdog/src/main/java/co/cask/cdap/logging/read/KafkaLogReader.java

```diff
@@ -44,6 +44,8 @@
 public class KafkaLogReader implements LogReader {
   private static final Logger LOG = LoggerFactory.getLogger(KafkaLogReader.class);
   private static final int KAFKA_FETCH_TIMEOUT_MS = 30000;
+  // Maximum events to read from Kafka for any call
+  private static final int MAX_READ_EVENTS_KAFKA = 10000;
```

Comment:
Initially I thought about making it configurable, but then I realized that if we are not able to find even one log event after scanning 10k records then increasing it to 20k may not make much difference. 

What do you say? I don't have a strong opinion either way.